### PR TITLE
curvebs support expand volume

### PR DESCRIPTION
**Pull request by jolly-sy**
Signed-off-by: jolly-sy <757050468@qq.com>

### What problem does this PR solve?

Issue Number: #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: src/tools/curve_tool_define.h [hunk 1]
```diff
@@ -58,12 +58,13 @@ const char kSnapshotCloneStatusCmd[] = "snapshot-clone-status";
...
 const char kScanStatusCmd[] = "scan-status";
 
-// NamesPaceTool相关命令
+// NameSpaceTool相关命令
 const char kGetCmd[] = "get";
 const char kListCmd[] = "list";
 const char kSegInfoCmd[] = "seginfo";
 const char kDeleteCmd[] = "delete";
 const char kCreateCmd[] = "create";
+const char kExpandCmd[] = "expand";
```
Comment (wu-hanqing) [inline src/tools/curve_tool_define.h:67]: use `Extend`

File: src/tools/namespace_tool.cpp [hunk 2]
```diff
@@ -144,6 +148,8 @@ void NameSpaceTool::PrintHelp(const std::string &cmd) {
         std::cout << "expireTime: s=second, m=minute, h=hour, d=day, M=month, y=year" << std::endl;  // NOLINT
     } else if (cmd == kCreateCmd) {
         std::cout << "curve_ops_tool " << cmd << " -fileName=/test -userName=test -password=123 -fileLength=20‬  [-mdsAddr=127.0.0.1:6666] [-confPath=/etc/curve/tools.conf]" << std::endl;  // NOLINT
+    } else if (cmd == kExpandCmd) {
+        std::cout << "curve_ops_tool " << cmd << " -fileName=/test -userName=test -password=123 -newSize=30  [-mdsAddr=127.0.0.1:6666] [-confPath=/etc/curve/tools.conf]" << std::endl;  // NOLINT
```
Comment (wu-hanqing) [inline src/tools/namespace_tool.cpp:152]:
```suggestion
        std::cout << "curve_ops_tool " << cmd << " -fileName=/test -userName=test -password=123 -newSize(GiB)=30  [-mdsAddr=127.0.0.1:6666] [-confPath=/etc/curve/tools.conf]" << std::endl;  // NOLINT
```

File: src/tools/curve_tool_main.cpp [hunk 3]
```diff
@@ -45,6 +45,7 @@ const char* kHelpStr = "Usage: curve_ops_tool [Command] [OPTIONS...]\n"
         "delete : delete the file, to force delete, should specify the --forcedelete=true\n"  //NOLINT
         "clean-recycle : clean the RecycleBin\n"
         "create : create file, file length unit is GB\n"
+        "expand : expand volume of file\n"
```
Comment (ilixiaocui) [inline src/tools/curve_tool_main.cpp:48]: extend

File: src/tools/mds_client.cpp [hunk 4]
```diff
@@ -295,6 +295,28 @@ int MDSClient::CreateFile(const std::string& fileName, uint64_t length) {
...
+    FillUserInfo(&request);
+    curve::mds::CurveFSService_Stub stub(&channel_);
+    auto fp = &curve::mds::CurveFSService_Stub::ExtendFile;
+    if (SendRpcToMds(&request, &response, &stub, fp) != 0) {
+        std::cout << "extendFile(expandVolume) from all mds fail!" << std::endl;
+        return -1;
+    }
+
+    if (response.has_statuscode() &&
+                response.statuscode() == StatusCode::kOK) {
```
Comment (ilixiaocui) [inline src/tools/mds_client.cpp:312]: std::cout << extend ok ***

**Pull request closed without merging**